### PR TITLE
Migrate kubernetes yaml's to helm chart

### LINKED ISSUE #313: Helm support
Hi guys,
You've done a great job with this project! Amazing work.

Are you planning to add Helm? If so, I could help you to organize the charts.

Btw. did you manage to run v0.4.6 with Kubernetes?
Every time one joins a session, then "stream removed" event appears after few seconds. It works fine with docker-compose (locally). I had my cluster running on Digital Ocean.

### PR DESCRIPTION
**Pull request by billylindeman**
I have converted the yaml config from `kube/` into a helm chart.  I have been testing it in Google Kubernetes Engine and it seems to work, though I suspect it will need a bit more configuration and tweaking before its completely ready.

fixes #313

### CODE REVIEW

File: kube/ion/templates/deployment-sfu.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      containers:
+        - name: {{ .Chart.Name }}-sfu
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          image: pionwebrtc/ion:v0.4.6
+          command: ["/usr/local/bin/sfu", "-c", "/configs/sfu.toml"]
+          resources:
+            requests:
+              cpu: 250m
```
Comment (leewardbound) [inline kube/ion/templates/deployment-sfu.yaml]: I know this figure came from me, just making a note -- I am certain I winged these figures originally, we will want to get a second opinion on these defaults (and/or find good practical limits via load testing), but this is sensible enough to commit; we have a strong "alpha quality" warning so I am gonna give this a :heavy_check_mark:

File: kube/ion/values.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# Default values for ion.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
```
Comment (leewardbound) [inline kube/ion/values.yaml]: @billylindeman might be overkill, but i suggest adding a heavy paragraph disclaimer here -- "SFU relay is WIP, do not scale beyond 1 SFU" etc, as a lot of questions come from when people dive into deploying the k8s charts without consulting the readme

**Review by leewardbound — COMMENTED** [review]
I'm cool with this, gonna ask for a review from @jbrady42 first because he is definitely better k8s than me (and @tarrencev i know you mentioned wanting to try this too)

**Review by leewardbound — APPROVED** [review]

File: kube/ion/templates/deployment-sfu.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      {{- end }}
+      serviceAccountName: {{ include "ion.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}-sfu
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          image: pionwebrtc/ion:v0.4.6
```
Comment (m9rc1n) [inline kube/ion/templates/deployment-sfu.yaml]: @billylindeman 

This line is missing `-sfu`
Should be `pionwebrtc/ion-sfu:v0.4.6`

Comment (billylindeman) [reply kube/ion/templates/deployment-sfu.yaml]: ah good catch! I find/replaced some stuff and this must've gotten caught in it

**Review by jbrady42 — CHANGES_REQUESTED** [review]
This is cool to have templated out in helm. Though honestly I prefer to work directly with the raw yaml files and potentially other tools like kustomize. We should leave the stock kube config in native yaml in another dir as an option and allow users to choose whichever they prefer.

**Comment by leewardbound** [comment]
@billylindeman is this ready to merge? I'm good with it. @jbrady42 are we good?

**Review by jbrady42 — APPROVED** [review]
:+1:

**Pull request merged**